src/app/learn/reference/faq/index.html

```diff
@@ -22,7 +22,7 @@
   </stache-page-anchor>
 
   <p>
-    In the <stache-code>package.json</stache-code> configuration file for NPM that the SKY UX template provides in the default SPA, update the <stache-code>@blackbaud/skyux</stache-code> property that specifies the version of SKY UX components. To find the latest version of the SKY UX components, check the <a href="https://github.com/blackbaud/skyux2/blob/master/CHANGELOG.md">changelog in the SKY UX repo</a> or the <a href="https://www.npmjs.com/package/@blackbaud/skyux">NPM page for the SKY UX package</a>.
+    To update the packages and modules that you install as dependencies in your project, run <stache-code>skyux upgrade</stache-code>. This command pulls in the latest version of the <stache-code>devDependencies</stache-code> and <stache-code>peerDependencies</stache-code> as well as any dependencies listed in the SPA's <stache-code>package.json</stache-code>. For more information on how to install and use SKY UX modules, <a routerLink="/learn/get-started/basics/modules">use SKY UX modules</a>. To find the latest version of the SKY UX components, check the <a routerLink="/contribute/packages">changelog in the component's repository</a> or the <a href="https://www.npmjs.com/package/@blackbaud/skyux">NPM page for the SKY UX package</a>.
```

Comment:
I don't think we need the last sentence. I think we should delete it.

Also, in the third sentence that you added linking to https://developer.blackbaud.com/skyux/learn/get-started/basics/modules, we're missing "see" after the comma, and the repeated "use SKY UX modules" reads awkwardly. I'd change that sentence to "For more information on how to install and update modules, see use SKY UX modules."